.github/dependabot.yml

```diff
@@ -25,6 +25,10 @@ updates:
   directory: "/tools/docs"
   schedule:
     interval: "daily"
+  ignore:
+    # Losing bullet lists and need to keep pinned to 0.16.
+    # See https://stackoverflow.com/questions/67542699/readthedocs-sphinx-not-rendering-bullet-list-from-rst-file
+    - dependency-name: "docutils"
```

Comment:
i dont think we want this if we only want to prevent one version upgrade

if you remove this and land the pr, it will create a pr to upgrade, if you close that pr it will not do so again until there is a newer dep

Reply:
also - the way im currently managing updates that we dont want...

i add a PR for the blocked updates - see eg #16248 with `[BLOCKED] Update foo` in the title

then, as any new updates come in i either check upstream or check ci to see if whatever was blocking it is fixed

this way we track what is broken but handle updates as they come

Reply:
OK sounds good I will remove this.

Reply:
k - if we remove this - lgtm - i can follow up the rest